### DISCUSSION TITLE
Light nvim version to use as a git editor

### DISCUSSION THREAD

**Discussion by golergka** [LazyVim/LazyVim#2307] [Q&A] [answered]
I use lazyvim as my primary IDE, which involves a lot of plugins. But I also have it configured as `core.editor` in my git configuration, so it starts up every time I need to edit a commit message or a patch. It doesn't start up that fast, and ligatures I configured for GUI mode don't always show up correctly. Is there a way to start up nvim in "light" mode with a lot of features and plugins disabled which would be suitable for terminal one-file use?

**Comment by dpetka2001** [comment, marked as answer]
You can put in your `core.editor = nvim --clean` and it will start with no plugins whatsoever.
